js/popup-editor.js

```diff
@@ -0,0 +1,145 @@
+import { isEnterEvent, isEscapeEvent, validateHashtag, checkLengthLine } from './utils.js';
+
+const MAX_LENGTH_COMMENT = 140;
+
+const body = document.querySelector('body');
+const formDownloadPicture = document.querySelector('#upload-select-image'); //форма, собирающая все данные про изображение
+const inputFilePicture = formDownloadPicture.querySelector('#upload-file'); //поле загрузки фото
+//const buttonSubmit = formDownloadPicture.querySelector('#upload-submit'); //кнопка, отправляет форму
+
+const popupEditor = formDownloadPicture.querySelector('.img-upload__overlay'); //окно редактора загруженной картинки
+const inputHashtag = popupEditor.querySelector('#input-hashtag'); //поле ввода хэштега
+const inputComment = popupEditor.querySelector('#input-comment'); //поле ввода коммента
+const buttonCloseEditor = popupEditor.querySelector('#upload-cancel'); //кнопка, закрывает редактор
+
+let validationHashtagsIsOk = false;
```

Comment:
```suggestion
let isHashtagsValid = false;
```
молодец, что додумалась до `is`!